modules/customize-certificates-replace-default-router.adoc

```diff
@@ -14,6 +14,13 @@ and CLI, will have encryption provided by specified certificate.
 
 * You must have a wildcard certificate and its private key,
 both in the PEM format, for use.
++
+[WARNING]
+====
+The new certificate *must* be signed by a public certificate authority already
+included in the CA bundle as provided by the container userspace.
```

Comment:
The cert does not need to be signed by a public ca. The documented procedures are for adding a custom, i.e. non-public, ca. If the cert is signed by a public ca, then only the secret and ingresscontroller steps are required.

Reply:
@danehans thank you for clarifying that. After a careful second reread, I see what you mean. Since this tripped up an external user and myself, it's probably helpful to make this overly obvious; I've modified this PR to add a note for pointing this fact out so it's not overlooked. Can you confirm?